# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/linux/hotfix-kern-cont-aha152x.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- linux-6.8/drivers/scsi/aha152x.c.vanilla	2024-05-07 00:51:23.119298807 +0200
+++ linux-6.8/drivers/scsi/aha152x.c	2024-05-07 00:57:34.281328460 +0200
@@ -833,7 +833,7 @@
 				"Please verify.\n", shpnt->host_no, shpnt->irq);
 		goto out_host_put;
 	}
-	printk("ok.\n");
+	printk(KERN_CONT "ok.\n");
 
 
 	/* clear interrupts */
